pkg/operator/operator.go

```diff
@@ -123,11 +121,11 @@ func New(
 		operandVersions: operandVersions,
 	}
 
-	deployInformer.Informer().AddEventHandler(optr.eventHandlerDeployments())
-	daemonsetInformer.Informer().AddEventHandler(optr.eventHandler())
-	validatingWebhookInformer.Informer().AddEventHandler(optr.eventHandlerSingleton(isMachineWebhook))
-	mutatingWebhookInformer.Informer().AddEventHandler(optr.eventHandlerSingleton(isMachineWebhook))
-	featureGateInformer.Informer().AddEventHandler(optr.eventHandler())
+	_, _ = deployInformer.Informer().AddEventHandler(optr.eventHandlerDeployments())
```

Comment:
Hm, I'm not quite sure why is this needed...

@slintes were there some issues with linter without this?

Reply:
yes indeed, linter was complaining about unevaluated retirn values